src/components/login/LoginForm.vue

```diff
@@ -45,6 +45,12 @@
                 Password is required
             </p>
         </div>
+        <div>
+            <p class="card__subtitle">
+                You don't have an account yet ?
+                <a class="card__action" href="#/register">Create one here !</a>
```

Comment:
Tu peux utiliser la balise `router-link`. ca permet de ne pas recharger la page, c'est le router qui prend en charge le chargement de la vue. Ca pourrait donner ça:
`<router-link to="/register" class="card__action">Create one here !</router-link>`